docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,237 @@
+    from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import OTLPMetricExporter
+    from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+    from opentelemetry.sdk._logs import LoggerProvider, LoggingHandler
+    from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+    from opentelemetry.sdk.metrics import MeterProvider
+    from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+    def configure_oltp_grpc_tracing(
+        endpoint: str = None
+    ) -> trace.Tracer:
+        # Configure Tracing
+        traceProvider = TracerProvider()
+        processor = BatchSpanProcessor(OTLPSpanExporter(endpoint=endpoint))
+        traceProvider.add_span_processor(processor)
+        trace.set_tracer_provider(traceProvider)
+
+        # Configure Metrics
+        reader = PeriodicExportingMetricReader(OTLPMetricExporter(endpoint=endpoint))
+        meterProvider = MeterProvider(metric_readers=[reader])
+        metrics.set_meter_provider(meterProvider)
+
+        # Configure Logging
+        logger_provider = LoggerProvider()
+        set_logger_provider(logger_provider)
+
+        exporter = OTLPLogExporter(endpoint=endpoint)
+        logger_provider.add_log_record_processor(BatchLogRecordProcessor(exporter))
+        handler = LoggingHandler(level=logging.NOTSET, logger_provider=logger_provider)
+        handler.setFormatter(logging.Formatter("Python: %(message)s"))
+
+        # Attach OTLP handler to root logger
+        logging.getLogger().addHandler(handler)
+
+        tracer = trace.get_tracer(__name__)
+        return tracer
+    ```
+
+1. Update your application (`app.py` for Flask, `main.py` for FastAPI) to include the imports and call the `configure_oltp_grpc_tracing` function:
+
+    ```python
+    import logging
+    from otlp_tracing import configure_otel_otlp
+
+    logging.basicConfig(level=logging.INFO)
+    tracer = configure_otel_otlp()
+    logger = logging.getLogger(__name__)
+    ```
+
+1. Replace the `print` calls with `logger.info` calls in your application.
+1. Restart your application.
+
+### Framework Specific Instrumentation
+
+This instrumentation has only focused on adding OpenTelemetry to our code. For more detailed instrumentation, you can use the OpenTelemetry Instrumentation packages for the specific frameworks that you are using.
+
+### [Flask](#tab/flask)
+
+1. Install the Flask instrumentation package:
+
+    ```Console
+    pip install opentelemetry-instrumentation-flask
+    ```
+
+1. Add the following code to your application:
+
+    ```Python
+    from opentelemetry.instrumentation.flask import FlaskInstrumentor
+
+    # add this line after configure_otel_otlp() call
+    FlaskInstrumentor().instrument()
+    ```
+
+### [FastAPI](#tab/fastapi)
+
+1. Install the FastAPI instrumentation package:
+
+    ```Console
```

Comment:
```suggestion
    ```python
```

Reply:
`pip` is a CLI tool (like NuGet), which is why it's `Console` (or should be `console`).

```suggestion
    ```console
```